src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private static final int TEN = 10;
```

Comment:
Not quite the correct name. The fact that it is 10 is already visible, but why 10, what is it?